packages/@tinacms/fields/src/components/TextField.ts

```diff
@@ -29,6 +29,6 @@ export interface TextFieldProps extends a {
   ref?: any
 }
 
-export const TextField = styled.input<TextFieldProps>`
+export const BaseTextField = styled.input<TextFieldProps>`
```

Comment:
I don't think this should have been changed

Reply:
The `TextFieldPlugin` module exports `TextField` as well. This is just a WIP but there are a few of these types of collisions that I need to think about

Reply:
oof 🤔